components/nexus-base/src/main/java/org/sonatype/nexus/internal/log/LogbackLoggerOverrides.java

```diff
@@ -62,7 +62,7 @@
   @Inject
   public LogbackLoggerOverrides(final ApplicationDirectories applicationDirectories) {
     checkNotNull(applicationDirectories);
-    this.file = new File(applicationDirectories.getWorkDirectory("etc/logback"), "logback-overrides.xml");
+    this.file = new File(applicationDirectories.getConfigDirectory("logback"), "logback-overrides.xml");
```

Comment:
The original line is correct. `logback-overrides.xml` is a mutable file, generated and updated according to changes made in the Nexus UI. Therefore it should be under the work directory, which is guaranteed to be mutable for the Nexus user, rather than the install directory which may be read-only.